ambry-router/src/main/java/com.github.ambry.router/NonBlockingRouterMetrics.java

```diff
@@ -105,6 +105,16 @@
   public Gauge<Long> requestResponseHandlerThreadRunning;
   public Gauge<Integer> activeOperations;
 
+  // metrics for tracking blob sizes and chunking.
+  public final Histogram putBlobSizeBytes;
+  public final Histogram putBlobChunkCount;
+  public final Histogram getBlobSizeBytes;
+  public final Histogram getBlobChunkCount;
+  public final Counter simpleBlobPutCount;
```

Comment:
Just want to note here that even `Meter` has an ever increasing count but also exports a rate - so in essence it provides the info of a `Counter` + some extra data. Personally I think `Counter` is right here.